tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -92,22 +100,18 @@ def example_plot(ax, fontsize=12, hide_labels=False):
 for ax in axs.flat:
     example_plot(ax)
 
-# %%
-# Colorbars
+# %% Colorbars
```

Comment:
I don’t think this change was intended.  This title is missing in the rendered docs.